src/screens/tenants/CreateTenantDialog.tsx

```diff
@@ -27,7 +27,7 @@ interface State {
 export default class CreateTenantDialog extends React.Component<Props, State> {
   public state: State;
   public props: Props;
-  public tenantEndpointClouds: EndpointCloud[];
+  public tenantEndpointClouds: EndpointCloud[] = [];
```

Comment:
For the sake of consistency, you have to init all attributes in the constructor.